### PR TITLE
Account request v9 migration

### PR DESCRIPTION
**Pull request by kevin9foong**
**Outline of Solution**
Implemented the AccountRequest entity, UT and IT for the v9 migration. 

Tests: 
The tests have been fully written. 
* unit tests: create, update and delete  
* integration tests: create, different reads, update and delete

### CODE REVIEW

File: src/main/java/teammates/storage/sqlapi/AccountRequestDb.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    }
+
+    /**
+     * Creates an AccountRequest in the database.
+     */
+    public AccountRequest createAccountRequest(AccountRequest accountRequest)
+            throws InvalidParametersException, EntityAlreadyExistsException {
+        assert accountRequest != null;
+
+        accountRequest.sanitizeForSaving();
```
Comment (samuelfangjw) [inline src/main/java/teammates/storage/sqlapi/AccountRequestDb.java:42]: We have moved the sanitisation methods to the setters instead.

File: src/main/java/teammates/storage/sqlapi/AccountRequestDb.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        }
+
+        persist(accountRequest);
+        return accountRequest;
+    }
+
+    /**
+     * Get AccountRequest by {@code accountRequestId} from database.
+     */
+    public AccountRequest getAccountRequest(int accountRequestId) {
```
Comment (samuelfangjw) [inline src/main/java/teammates/storage/sqlapi/AccountRequestDb.java:60]: We may not need this now, but will change to mainly use id once everything has been migrated.

Comment (kevin9foong) [reply src/main/java/teammates/storage/sqlapi/AccountRequestDb.java:60]: I have removed this method for now.

File: src/main/java/teammates/storage/sqlapi/AccountRequestDb.java (hunk 1 above)
Comment (kevin9foong) [reply src/main/java/teammates/storage/sqlapi/AccountRequestDb.java:42]: I have moved the sanitisation methods to the setters.

**Pull request merged**